<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinuxAttrView</class>
 <widget class="QWidget" name="LinuxAttrView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>216</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hboxLsAttr" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="lblLsAttrDesc">
       <property name="text">
        <string>lsattr:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblLsAttr">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string notr="true">----------------------</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspcLsAttr">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkAppendOnly">
       <property name="toolTip">
        <string>File can only be opened in append mode for writing.</string>
       </property>
       <property name="text">
        <string>a: append only</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="chkTopDir">
       <property name="toolTip">
        <string>Directory will be treated like a top-level directory by the ext3/ext4 Orlov block allocator.</string>
       </property>
       <property name="text">
        <string>T: top dir</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="chkNoCompress">
       <property name="toolTip">
        <string>File is excluded from compression.</string>
       </property>
       <property name="text">
        <string>m: no compress</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="chkDAX">
       <property name="toolTip">
        <string>Direct access</string>
       </property>
       <property name="text">
        <string>x: DAX</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chkImmutable">
       <property name="toolTip">
        <string>File cannot be modified, deleted, or renamed.</string>
       </property>
       <property name="text">
        <string>i: immutable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chkDirSync">
       <property name="toolTip">
        <string>Changes to this directory are written synchronously to the disk.</string>
       </property>
       <property name="text">
        <string>D: dir sync</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="chkJournalled">
       <property name="toolTip">
        <string>File data is written to the journal before writing to the file itself.</string>
       </property>
       <property name="text">
        <string>j: journalled</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="chkIndexed">
       <property name="toolTip">
        <string>Directory is indexed using hashed trees.</string>
       </property>
       <property name="text">
        <string>I: indexed</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="chkFileSync">
       <property name="toolTip">
        <string>Changes to this file are written synchronously to the disk.</string>
       </property>
       <property name="text">
        <string>S: sync</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="chkCompressed">
       <property name="toolTip">
        <string>File is compressed.</string>
       </property>
       <property name="text">
        <string>c: compressed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="chkExtents">
       <property name="toolTip">
        <string>File is mapped on disk using extents.</string>
       </property>
       <property name="text">
        <string>e: extents</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkNoDump">
       <property name="toolTip">
        <string>This file is not a candidate for dumping with the dump(8) program.</string>
       </property>
       <property name="text">
        <string extracomment="&quot;dump&quot; is the name of the executable, so it should not be localized.">d: no dump</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chkCasefold">
       <property name="toolTip">
        <string>Files stored in this directory use case-insensitive filenames.</string>
       </property>
       <property name="text">
        <string>F: casefold</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="chkNoATime">
       <property name="toolTip">
        <string>Access time record is not modified.</string>
       </property>
       <property name="text">
        <string>A: no atime</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chkInlineData">
       <property name="toolTip">
        <string>File data is stored inline in the inode.</string>
       </property>
       <property name="text">
        <string>N: inline data</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QCheckBox" name="chkUndelete">
       <property name="toolTip">
        <string>File's contents will be saved when deleted, potentially allowing for undeletion. This is known to be broken.</string>
       </property>
       <property name="text">
        <string>u: undelete</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="chkProject">
       <property name="toolTip">
        <string>Directory will enforce a hierarchical structure for project IDs.</string>
       </property>
       <property name="text">
        <string>P: project</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="chkVerity">
       <property name="toolTip">
        <string>File has fs-verity enabled.</string>
       </property>
       <property name="text">
        <string>V: fs-verity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="chkEncrypted">
       <property name="toolTip">
        <string>File is encrypted.</string>
       </property>
       <property name="text">
        <string>E: encrypted</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="chkNoCOW">
       <property name="toolTip">
        <string>Not subject to copy-on-write updates.</string>
       </property>
       <property name="text">
        <string>C: no CoW</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="chkNoTailMerge">
       <property name="toolTip">
        <string>If the file system supports tail merging, this file will not have a partial block fragment at the end of the file merged with other files.</string>
       </property>
       <property name="text">
        <string>t: no tail merge</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="chkSecureDelete">
       <property name="toolTip">
        <string>File's blocks will be zeroed when deleted.</string>
       </property>
       <property name="text">
        <string>s: secure del</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="hspcGrid">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
